scenarios/node_hog_rbac.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Is this file still needed since the config is now using the URL?

Reply:
Should we keep the experiment file (had previously deleted from this PR) and this file for the disconnected clusters situation you were talking about in your previous comment? Or better to just call it out in the documentation that you'll have to download and use the files of whatever version they want so it doesn't become super out of date?

Reply:
That was my confusion, I needed to down load the experiment file and add it to config.yaml (it is gone from there too).

-  wget https://hub.litmuschaos.io/api/chaos/1.9.1?file=charts/generic/node-cpu-hog/experiment.yaml -O scenarios/experiment.yaml
- edit config/config.yml and add experiment.yaml after node_hog_engine.yaml
- TARGET_NODES worked fine for 1.9.1 - that was in the sample experiment file 

Our sample config.yaml should probably work OOTB and include a local experiment file and local engine file since they are most likely to be updated.  If not, we need to include instructions in the Litmus README for downloading scenarios and adding them to the config.   My opinion, anyways.    @jordigilh @chaitanyaenr other opinions.

Reply:
From my understanding, the code will install all experiments based on the version you specify in the overarching config file. So no need to do the wget or have the experiment files locally. 
Setting the TARGET_NODES in ChaosEngine file should set an environment variable to properly set the variables for the specific experiment file with the given target node name.

Working on validating this and updating the documentation file for litmus to be a little more clear on the steps needed.

Reply:
Yes, you can keep this locally for disconnected clusters, but then you'll have to make sure that the spec matches the litmus go runner version, particularly when you upgrade the litmus version.
You only need to apply the `chaosengine` that will be used for this particular experiment, and discard the others.

Reply:
Removed the rbac file for the node_hog scenario

Reply:
I think you'll need the rbac file to define the sa and the role an role binding. Unless you are handling that elsewhere :)

Reply:
I meant removed the downloaded rbac file from the scenarios folder. Still using the hub.litmus link in the config to set the sa, role and role binding

Reply:
ok sorry, my bad :)